packages/contract-addresses/addresses.json

```
@@ -20,6 +20,7 @@
         "devUtils": "0x74134cf88b21383713e096a5ecf59e297dc7f547",
         "erc20BridgeProxy": "0x8ed95d1746bf1e4dab58d8ed4724f1ef95b20db0",
         "uniswapBridge": "0x36691c4f426eb8f42f150ebde43069a31cb080ad",
+        "uniswapV2Bridge": "0x0000000000000000000000000000000000000000",
```

```suggestion
        "uniswapV2Bridge": "0xdcd6011f4c6b80e470d9487f5871a0cba7c93f48",
```

Deployed https://etherscan.io/address/0xdcd6011f4c6b80e470d9487f5871a0cba7c93f48#code